testing/clean.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+find ./tmp/* ! -name '*.keep' -delete
```

Comment:
I'd use `find -mindepth 1 tmp ! -name '*.keep' -delete`, to avoid an error should `./tmp` be empty, but that's just nitpicking.